packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -1417,6 +1419,24 @@ class ObjectGroup extends ObjectGroupBase {
     final directories = (invocationResult as List?)?.cast<Object>();
     return List.from(directories ?? []);
   }
+
+  @override
+  Future<Object?> screenShot(InspectorInstanceRef selection) {
+    if (disposed) {
+      return Future.value();
+    }
+    if (useDaemonApi) {
+      return invokeServiceMethodDaemonParams('screenshot', {
+        'id': selection.id,
+        'width': 480,
```

Comment:
pass in width and height as named parameters to this method.
I would also recommend using sizes based on the size available in the UI rather than forcing to 480 and 640. Smaller screenshots are significantly faster to send from the device and to encode.

Reply:
yes you are right i will revise it.